src/main/kotlin/calculator/StringDelimiterSeparator.kt

```diff
@@ -0,0 +1,36 @@
+package calculator
+
+@JvmInline
+value class StringDelimiterSeparator(private val delimiterRegex: Regex) {
+
+    init {
+        val pattern = delimiterRegex.pattern
+        require(pattern.contains("<$DELIMITER_GROUP_NAME>") and pattern.contains("<$STRINGS_GROUP_NAME>")) { "$DELIMITER_GROUP_NAME and $STRINGS_GROUP_NAME must be included in regex pattern. but provided pattern(`$delimiterRegex`)" }
```

Comment:
```suggestion
        val pattern = delimiterRegex.pattern
        require(
            pattern.contains("<$DELIMITER_GROUP_NAME>") and pattern.contains("<$STRINGS_GROUP_NAME>")
        ) {
            "$DELIMITER_GROUP_NAME and $STRINGS_GROUP_NAME must be included in regex pattern. " +
                "but provided pattern(`$delimiterRegex`)"
        }
```
가로로 길어지는 경우에는 적절하게 개행해주시면 가독성에 큰 도움이 됩니다.

Reply:
넵 감사합니다!! 
에러 메세지에 대해서는 신경쓰지 못한 부분이었습니다!!